Example/Example/AppDelegate.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+import SalemoveSDK
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    let salemoveDelegate = SalemoveAppDelegate()
+}
+
+// MARK: Initialization
+extension AppDelegate {
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+
+        salemoveDelegate.application(application, didFinishLaunchingWithOptions: launchOptions)
+
+        configureSalemove()
+
+        return true
+    }
+}
+
+extension AppDelegate {
+    func configureSalemove() {
+        try? Salemove.sharedInstance.configure(site: "72bd8093-eae1-4f31-88c1-0c3132eb0906")
+        try? Salemove.sharedInstance.configure(environment: "https://api.salemove.com/")
+        try? Salemove.sharedInstance.configure(appToken: "sEQaZ6S2ULiFGK9F")
```

Comment:
Who's credentials are those?

Reply:
I'd like to remind that this project *is public*.

Reply:
@DmitryPR now i understand what you meant by including credentials in the app. Perhaps we can read them from a conf file that is not public?

Reply:
Do we need to include them at all? Clients can put their own credentials if they want to run the app.

Reply:
These are same credentials as in the integration guide
https://github.com/salemove/ios-bundle/blob/master/2017-11-28-Enhancing%2BYour%2BiOS%2BApplication%2BWith%2BSalmove%2BSDK.md

Reply:
Are they working? If they are not working then perhaps just replace them with some "YOUR_SITE", "YOUR_APP_TOKEN".